Library/include/CSP/Systems/Quota/Quota.h

```diff
@@ -124,6 +139,9 @@ class CSP_API FeaturesLimitResult : public csp::services::ResultBase
 	/// @return csp::common::Array<FeatureProgress> : const array of feature progress class
 	const csp::common::Array<FeatureLimitInfo>& GetFeaturesLimitInfo() const;
 
+protected:
+	int ParseErrorCode(const csp::common::String& Value) override;
```

Comment:
Would it be ok to add the `virtual` qualifier to these overrrides? For me personally, it helps a little with readability.

Reply:
sorted

Reply:
We should avoid using virtual where we don't want functions to be overridden by further derived classes. As there should never be a need for us to derive from `FeaturesLimitResult`, `ParseErrorCode` should only be marked as `override` and not `virtual`.

Reply:
Yep all good, happy with that